extensions/example-extension/renderer.tsx

```diff
@@ -30,7 +30,7 @@ export default class ExampleExtension extends LensRendererExtension {
     }
   ];
 
-  clusterPageMenus: Interface.PageMenuRegistration[] = [
+  clusterPageMenus: Interface.ClusterPageMenuRegistration[] = [
```

Comment:
this should be identified as a breaking change?

Reply:
Probably

Reply:
Defining interfaces was totally optional, so 50/50

Reply:
It is a breaking change since a name was changed, the easy solution is to do the following:

```typescript
/**
 * @depreciated use ClusterPageMenuRegistration
 */
export type PageMenuRegistration = ClusterPageMenuRegistration;
```

Reply:
Wait, it's not removed, `PageMenuRegistration` exists.. They are both exported:

```ts
export interface PageMenuRegistration {
  target?: PageTarget;
  title: React.ReactNode;
  components: PageMenuComponents;
}

export interface ClusterPageMenuRegistration extends PageMenuRegistration {
  id?: string;
  parentId?: string;
}
``

Reply:
Ah I see, then it certainly isn't a breaking change. My bad.

Reply:
just by virtue of the fact that `id` and `parentId` are optional, right?

Reply:
Yes basically